### PR TITLE
Rename/vbrListMethods to VBR_list_methods

### PR DESCRIPTION
**Pull request by krishchvn**
renamed vbrListMethods to VBR_list_methods in all files #109

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -144,26 +148,23 @@ VBR.out.viscous.HZK2011.eta_total % composite steady state creep viscosity
 
 The VBR Calculator nominally works in GNU Octave, but you may find that you need to install some packages.
```
Comment (chrishavlin) [inline README.md:149]:
```suggestion
The VBR Calculator works with [GNU Octave](https://octave.sourceforge.io/io/index.html) as well as MATLAB. Some of the `Projects/` may require additional packages. For how to install GNU Octave and additional packages, see:
* https://octave.sourceforge.io/io/index.html
* https://octave.org/doc/interpreter/Installing-and-Removing-Packages.html
```

File: README.md [hunk 2]
```diff
@@ -144,26 +148,23 @@ VBR.out.viscous.HZK2011.eta_total % composite steady state creep viscosity
...
 
-https://octave.sourceforge.io/io/index.html
-https://octave.sourceforge.io/statistics/index.html
-https://octave.org/doc/interpreter/Installing-and-Removing-Packages.html
+https://octave.sourceforge.io/io/index.html https://octave.sourceforge.io/statistics/index.html https://octave.org/doc/interpreter/Installing-and-Removing-Packages.html
```
Comment (chrishavlin) [inline README.md:151]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -144,26 +148,23 @@ VBR.out.viscous.HZK2011.eta_total % composite steady state creep viscosity
...
 
 # Getting Help
 
-See the full documentation page on [Contributing and Getting Help](https://vbr-calc.github.io/vbr/contrib/contributing/)
-for ways to reach out or [post a github issue](https://github.com/vbr-calc/vbr/issues/new)!
+See the full documentation page on [Contributing and Getting Help](https://vbr-calc.github.io/vbr/contrib/contributing/) for ways to reach out or [post a github issue](https://github.com/vbr-calc/vbr/issues/new)!
```
Comment (chrishavlin) [inline README.md:155]:
```suggestion
For bug reports, please open a [github issue](https://github.com/vbr-calc/vbr/issues/new). For general questions, join the VBRc slack channel and ask in the #vbr-help channel (join the slack from the invite link on the [Contributing and Getting Help](https://vbr-calc.github.io/vbr/contrib/contributing/) page).
```

File: README.md [hunk 4]
```diff
@@ -144,26 +148,23 @@ VBR.out.viscous.HZK2011.eta_total % composite steady state creep viscosity
...
-* create your fork of the VBRc repo
-* checkout a new branch
-* do work on your new branch
-* push those changes to your fork on github
-* submit a pull request back to the main VBRc repo
+- create your fork of the VBRc repo
+- checkout a new branch
+- do work on your new branch
+- push those changes to your fork on github
+- submit a pull request back to the main VBRc repo
```
Comment (chrishavlin) [inline README.md:167]: I'm guessing your IDE auto-changed this. I'd rather keep the * syntax for the unorded list though.
```suggestion
* create your fork of the VBRc repo
* checkout a new branch
* do work on your new branch
* push those changes to your fork on github
* submit a pull request back to the main VBRc repo
```

File: docs/_pages/gettingstarted/quickstart.md [hunk 5]
```diff
@@ -103,42 +105,38 @@ When changing parameters from those loaded by default, you can either load all t
 
 ## 5. Run the VBR Calculator
 
-The VBR Calculator executes calculations by passing the ```VBR``` structure to the ``VBR_spine()```:
+The VBR Calculator executes calculations by passing the `VBR` structure to the ``VBR_spine()```:
```
Comment (chrishavlin) [inline docs/_pages/gettingstarted/quickstart.md:108]:
```suggestion
The VBR Calculator executes calculations by passing the `VBR` structure to the `VBR_spine()`:
```

File: docs/_pages/vbrmethods/anelastic.md [hunk 6]
```diff
@@ -1,19 +1,16 @@
...
+The anelastic methods displayed by `VBR_list_methods` currently include:
 
-* `eburgers_psp` [documentation](/vbr/vbrmethods/anel/eburgerspsp/): Extended Burgers Model, pseudo period scaling
-* `andrade_psp` [documentation](/vbr/vbrmethods/anel/andradepsp/): Andrade Model, pseudo period scaling
-* `xfit_mxw` [documentation](/vbr/vbrmethods/anel/xfitmxw/): Master Curve Fit, maxwell scaling
-* `xfit_premelt` [documentation](/vbr/vbrmethods/anel/xfitpremelt/): Master Curve Fit, pre-melting maxwell scaling
+- `eburgers_psp` [documentation](/vbr/vbrmethods/anel/eburgerspsp/): Extended Burgers Model, pseudo period scaling
+- `andrade_psp` [documentation](/vbr/vbrmethods/anel/andradepsp/): Andrade Model, pseudo period scaling
+- `xfit_mxw` [documentation](/vbr/vbrmethods/anel/xfitmxw/): Master Curve Fit, maxwell scaling
+- `xfit_premelt` [documentation](/vbr/vbrmethods/anel/xfitpremelt/): Master Curve Fit, pre-melting maxwell scaling
```
Comment (chrishavlin) [inline docs/_pages/vbrmethods/anelastic.md:12]:
```suggestion
* `eburgers_psp` [documentation](/vbr/vbrmethods/anel/eburgerspsp/): Extended Burgers Model, pseudo period scaling
* `andrade_psp` [documentation](/vbr/vbrmethods/anel/andradepsp/): Andrade Model, pseudo period scaling
* `xfit_mxw` [documentation](/vbr/vbrmethods/anel/xfitmxw/): Master Curve Fit, maxwell scaling
* `xfit_premelt` [documentation](/vbr/vbrmethods/anel/xfitpremelt/): Master Curve Fit, pre-melting maxwell scaling
```

File: docs/_pages/vbrmethods/elastic.md [hunk 7]
```diff
@@ -1,13 +1,13 @@
...
 
-The elastic methods displayed by `vbrListMethods` currently include:
+The elastic methods displayed by `VBR_list_methods` currently include:
 
-* `anharmonic` [documentation](/vbr/vbrmethods/el/anharmonic/): anharmonic scaling
-* `anh_poro` [documentation](/vbr/vbrmethods/el/anhporo/): poro-elastic scaling
-* `SLB2005` [documentation](/vbr/vbrmethods/el/slb2005/): Stixrude and Lithgow‐Bertelloni (2005)
+- `anharmonic` [documentation](/vbr/vbrmethods/el/anharmonic/): anharmonic scaling
+- `anh_poro` [documentation](/vbr/vbrmethods/el/anhporo/): poro-elastic scaling
+- `SLB2005` [documentation](/vbr/vbrmethods/el/slb2005/): Stixrude and Lithgow‐Bertelloni (2005)
```
Comment (chrishavlin) [inline docs/_pages/vbrmethods/elastic.md:11]:
```suggestion
* `anharmonic` [documentation](/vbr/vbrmethods/el/anharmonic/): anharmonic scaling
* `anh_poro` [documentation](/vbr/vbrmethods/el/anhporo/): poro-elastic scaling
* `SLB2005` [documentation](/vbr/vbrmethods/el/slb2005/): Stixrude and Lithgow‐Bertelloni (2005)
```

**Review by chrishavlin — CHANGES_REQUESTED** [review]
Thanks for the PR @krishchvn !

A couple comments and to-do items:

1. Looks like whatever IDE you used applied a bunch of auto-formatting here. Most of the changes from that are fine, but in the future it's easier for me to review changes just related to the goal of PR. I did add code suggestions to undo some of those format changes and to fix a couple other typos I noticed while reading through the changes -- please go through and accept those code changes and make sure your IDE doesn't revert them when you pull the changes locally. 

2. I mentioned this in the issue #109 , but we don't want to outright delete the old `vbrListMethods.m` right away. If folks have been using that function, it makes a smoother transition to instead use a deprecation cycle. You've done the first step in this:  rename `vbrListMethods.m` to `VBR_list_methods.m`. But you also need to add a **new** `vbrListMethods.m` that prints a message and wraps the new `VBR_list_methods.m` as follows:

```matlab
function vbrListMethods(single_prop)
  %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
  % vbrListMethods
  %
  % DEPRECATED FUNCTION: This function is has been renamed to 
  %     VBR_list_methods() and vbrListMethods will be removed in a 
  %     future version of the VBRc.
  %
  % 
  % prints available methods by property to screen
  %
  % Parameters:
  % -----------
  % single_prop: optional string, must be in 'anelastic', 'elastic' or 'viscous'
  %
  % vbrListMethods() will print all methods
  % vbrListMethods('viscous') will print only viscous methods 
  %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
    msg = ['DEPRECATION WARNING: vbrListMethods has been renamed to ', ...
       'VBR_list_methods() and vbrListMethods will be removed in a ', ...
       'future version of the VBRc. Use VBR_list_methods to silence ', ...
       'this warning.'];
    disp(msg)
    if exist('single_prop', 'var')
        VBR_list_methods(single_prop)
    else
        VBR_list_methods()
    end 
end 
```

**Comment by krishchvn** [comment]
Hi Chris,

Thank you for explaining. As a beginner, it really helps as it's difficult to visualize git commands. 
I have committed your suggested changes and added a new function vbrListMethods() wrapping VBR_list_methods and showing a deprecation warning.
Please let me know if any changes will be required. 
Happy to contribute and learn!

**Comment by chrishavlin** [comment]
Great, thanks! I should be able to take a look later today. Thanks for you work on this!

File: vbr/support/vbrListMethods.m [hunk 8]
```diff
@@ -1,15 +1,15 @@
-function vbrListMethods(single_prop)
+function VBR_list_methods(single_prop)
```
Comment (chrishavlin) [inline vbr/support/vbrListMethods.m:1]: @krishchvn one more change here before it's ready... This `VBR_list_methods` function should be in it's own file, `VBR_list_methods.m`. If you just cut out what you've got here and leave the `vbrListMethods` below then it should be in a good state. Thanks!

Comment (chrishavlin) [reply vbr/support/vbrListMethods.m:1]: Just to be **super** clear, here's what I mean. We should have two files:

* `VBR_list_methods.m` containing the `VBR_list_methods` function that you have here. 
* `vbrListMethods.m` containing the `vbrListMethods` function that you have down below.

Comment (krishchvn) [reply vbr/support/vbrListMethods.m]: Hi. I'm so sorry I missed that out. 
I've done the appropriate changes.

**Review by chrishavlin — APPROVED** [review]
Thanks for the work on this, @krishchvn !

**Pull request merged**